spec/features/admin_adds_new_video_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+feature 'Admin adds new video' do
+  scenario "Admin successfully adds a new video" do
```

Comment:
I agree with you somewhat on this, however, since it is still a simple spec, I am going to leave it be.  You suggestion is noted.